lib/tizen_sdk.dart

```diff
@@ -112,10 +112,9 @@ class TizenSdk {
     String profile,
     @required String arch,
   }) {
-    final String type = arch == 'x86' ? 'emulator' : 'device';
     final String id = profile == null
-        ? 'wearable-$defaultTargetPlatform-$type.flutter'
-        : '${profile.replaceFirst('common', 'wearable')}-$type.flutter';
+        ? 'wearable-4.0-$arch.flutter'
+        : '${profile.replaceFirst('common', 'wearable')}-$arch.flutter';
```

Comment:
this substitution makes error regarding missing `rootstraps/*.flutter.xml` harder to understand, at least that was my experience
I'm not suggesting change here, I think I will submit an issue about the error message.

Reply:
I'm actually aware of the error and you can simply remove the broken links to work around. Sorry.

Reply:
Actually I couldn't find a very good way to remove those stale files automatically.

Reply:
I think I was referring to a different error regarding stale rootstraps. The error you are referring to must be generated in these conditions:

- The user tries to build a native plugin (or app) without installing a target platform SDK (e.g. Tizen 4.0 SDK for Tizen 4.0 app). In this case the user can just run `flutter-tizen doctor` and follow its instructions.
- The user tries to build a native plugin (or app) for unknown platform version (such as 6.5) or architecture.

In either case it would be better to properly warn the user of the missing rootstrap instead of printing error messages that are usually hard to understand.

Reply:
I improved the error messages when the target rootstrap definition is missing (for both cases in the above).